static/sass/_pattern_navigation.scss

```diff
@@ -53,14 +53,22 @@ $meganav-height: 3rem;
     }
 
     .p-navigation__nav {
-      margin-left: -1rem;
       margin-right: 0;
       max-height: calc(100vh - 3rem);
       width: auto;
 
       .p-navigation__items {
         width: 100%;
 
+        & > .p-navigation__item--dropdown-toggle > .p-navigation__link {
```

Comment:
It's probably out of scope for this PR, as it doesn't originate here, but the way the navigation is implemented here, with A LOT of overrides of very specific Vanilla class names is quite fragile and possibly problematic whenever we update something in Vanilla.

Variants like these ideally should be implemented with different class names from Vanilla. So instead of overriding `& > .p-navigation__item--dropdown-toggle > .p-navigation__link`, you would define `p-navigation__link--dense` (or whatever is the purpose of this) and use this new class name instead.

But as I mentioned, it's not just this one thing in this PR, the whole navigation seems to be written this way, so I mention this as maybe a potential for future improvements/refactoring.

Reply:
Ok, I have made a separate issue to tackle this as not to block this work https://warthogs.atlassian.net/browse/WD-8935